src/Nix/Builtins.hs

```diff
@@ -745,7 +753,7 @@ lessThan ta tb = ta >>= \va -> tb >>= \vb -> do
             (NInt   a, NFloat b) -> pure $ fromInteger a < b
             (NFloat a, NFloat b) -> pure $ a < b
             _ -> badType
-        (NVStr a _, NVStr b _) -> pure $ a < b
+        (NVStr a, NVStr b) -> pure $ a < b
```

Comment:
In the C++ code, `lessThan` only compares the underlying strings, not the contexts... what does `Ord` for `NixString` do?